tasks/gulp/compile-scss.js

```diff
@@ -40,7 +41,8 @@ gulp.task('scss:compile-accessible-autocomplete', () => gulp.src(`${configPaths.
   .pipe(plumber(errorHandler))
   .pipe(sourcemaps.init())
   .pipe(sass({
-    includePaths: ['node_modules'],
+    includePaths: ['node_modules', 'node_modules/govuk-frontend/dist', 'node_modules/govuk-frontend/dist/govuk/components'],
+    quietDeps: true,
```

Comment:
if we don't have quietDeps then we get a load of deprecation warnings from stuff imported from govuk-frontend (I think) https://design-system.service.mod.gov.uk/get-started/silence-deprecation-warnings-in-dart-sass/ think with this we should still get warned about deprecations in our sass, just not in stuff imported from node_modules